tests/TestCase/Database/Type/TimeTypeTest.php

```diff
@@ -216,31 +214,4 @@ public function testMarshal($value, $expected): void
             $this->assertSame($expected, $result);
         }
     }
-
-    /**
-     * Tests marshalling times using the locale aware parser
-     */
-    public function testMarshalWithLocaleParsing(): void
-    {
-        $expected = new DateTime('23:23:00');
-        $result = $this->type->useLocaleParser()->marshal('11:23pm');
-        $this->assertSame($expected->format('H:i'), $result->format('H:i'));
-        $this->assertNull($this->type->marshal('derp:23'));
-    }
-
-    /**
-     * Tests marshalling times in denmark.
-     */
-    public function testMarshalWithLocaleParsingDanishLocale(): void
-    {
-        $original = setlocale(LC_COLLATE, '0');
-        $updated = setlocale(LC_COLLATE, 'da_DK.utf8');
-        setlocale(LC_COLLATE, $original);
-        $this->skipIf($updated === false, 'Could not set locale to da_DK.utf8, skipping test.');
-
-        I18n::setLocale('da_DK');
-        $expected = new DateTime('03:20:00');
-        $result = $this->type->useLocaleParser()->marshal('03.20');
-        $this->assertSame($expected->format('H:i'), $result->format('H:i'));
```

Comment:
Do we not support locale-aware time parsing?

Reply:
Thought we don't need it since I didn't see timezone related stuff in the I18n class but re-added it now.